tests/wasmfs/wasmfs_create.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <assert.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+
+// FIXME: Merge with other existing close and open tests.
+
+int main() {
```

Comment:
It would also be good to test out-of-bounds reads, multiple reads and/or writes in a row (to make sure the offset is updated correctly), zero-sized reads/writes, and other interesting cases you can think of.

Reply:
I have a better test for testing out of bounds reads in the next PR.